spec/scrabble_spec.js

```diff
@@ -0,0 +1,65 @@
+var Scrabble = require('../scrabble');
+var Player = require('../player');
+
+describe('testing Scrabble module', function() {
+
+
+////////////// easy test :) //////////////////////
+    it('Scrabble can say hello', function() {
+      expect(Scrabble.helloWorld()).toBe("hello world!");
+    });
+
+/////////////////// score //////////////////////////
+    it('scrabble can score a word correctly', function() {
+        expect(Scrabble.score("word")).toBe(8);
+    });
+
+    it('scrabble can score a bingo correctly', function() {
+        expect(Scrabble.score("uuuuuuu")).toBe(57);
+    });
+
+    it('scrabble returns false when input is not a string', function() {
+        expect(Scrabble.score(23948)).toBe(false);
+        expect(Scrabble.score([3,5, "cat"])).toBe(false);
+    });
+
+    it('if a word does not have only letters, scrabble returns false', function() {
+        expect(Scrabble.score("23948cat")).toBe(false);
+        expect(Scrabble.score("cat0%")).toBe(false);
+    });
+
+/////////// highestScoreFrom working correctly ////////////
+    it('scrabble can score a list of words, none of which are bingos, correctly', function() {
+        expect(Scrabble.highestScoreFrom(["uuuuuu", "you", "cat", "syzygy" ])).toBe("syzygy");
+    });
+
```

Comment:
Bingo is a word that uses all 7 letters or more! Highly recommend this book. https://www.amazon.com/Word-Freak-Heartbreak-Obsession-Competitive/dp/0142002267